deploy/deploy.sh

```diff
@@ -59,6 +59,10 @@ metadata:
   name: ${SSH_BASTION_NAMESPACE}
   labels:
     openshift.io/run-level: "0"
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/audit: privileged
+    pod-security.kubernetes.io/warn: privileged
+    security.openshift.io/scc.podSecurityLabelSync=false
```

Comment:
should this have been 'security.openshift.io/scc.podSecurityLabelSync: false'?

I think we've got breakage: https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.12-upgrade-from-stable-4.11-e2e-azure-sdn-upgrade/1568121068034461696

```
 INFO[2022-09-09T06:59:06Z] Setting up ssh bastion
error: the server doesn't have a resource type "ssh-bastion"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  4298  100  4298    0     0  29438      0 --:--:-- --:--:-- --:--:-- 29438
+ set -e
+ SSH_BASTION_NAMESPACE=test-ssh-bastion
+ BASEDIR=https://raw.githubusercontent.com/eparis/ssh-bastion/master/deploy
+ trap clean_up EXIT
+ oc apply -f -
error: error parsing STDIN: error converting YAML to JSON: yaml: line 11: could not find expected ':' 
```